scripts/update_birdnet.sh

```diff
@@ -12,13 +12,12 @@ branch="main"
 while getopts ":r:b:" o; do
   case "${o}" in
     r)
-      remote=${OPTARG}
-
-      # Ensure that git remote is configured
-      git remote show $remote > /dev/null 2>&1 || {
+      if git remote show $remote &>/dev/null;then
```

Comment:
2 issues with this I'm seeing:
1. We need to set remote to argument first before this conditional checks for it in configured remotes
2. It's not working for me 
```bash
$ ./scripts/update_birdnet.sh -r bogus_remote_name -b remove-python-shebangs
+ sudo -u birder git reset --hard
HEAD is now at 24e20fb This needs a proper test to work
+ set +x
+ sudo -u birder git fetch bogus_remote_name remove-python-shebangs
fatal: 'bogus_remote_name' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
+ set +x
+ sudo -u birder git switch -C remove-python-shebangs --track bogus_remote_name/remove-python-shebangs
fatal: invalid reference: bogus_remote_name/remove-python-shebangs
+ set +x
ExecStart=/home/birder/BirdNET-Pi/birdnet/bin/python3 /usr/local/bin/server.py
ExecStart=/home/birder/BirdNET-Pi/birdnet/bin/python3 /usr/local/bin/daily_plot.py
```

Reply:
Please give [7a58b1c](https://github.com/mcguirepr89/BirdNET-Pi/pull/288/commits/7a58b1cc54e79607be7a0a653c37108c0833d22b) a test and let me know if this approach works for you.

Reply:
Yes that worked thank you for spotting that obvious mistake! of course it needs to be set before the check is meaningful :)